tests/CommunityToolkit.Mvvm.UnitTests/Test_ObservablePropertyAttribute.cs

```diff
@@ -1042,15 +1122,26 @@ public BroadcastingViewModel(IMessenger messenger)
         }
 
         [ObservableProperty]
-        [AlsoBroadcastChange]
+        [NotifyRecipients]
+        private string? name;
+    }
+
+    partial class RecipientWithNonBroadcastingProperty : ObservableRecipient
```

Comment:
Seems like we have a giant single file for all source generator tests? Should these be broken out into more files? one for base observable property modifies, one for recipients, one for validation at least?